pkgs/servers/monitoring/prometheus/varnish-exporter.nix

```diff
@@ -0,0 +1,24 @@
+# This file was generated by go2nix.
+{ stdenv, buildGoPackage, fetchgit, lib }:
+
+buildGoPackage rec {
+  name = "prometheus_varnish_exporter-${version}";
+  version = "20161126-${stdenv.lib.strings.substring 0 7 rev}";
+  rev = "218cffa79f9b56de6a21a9244d74bdc2ec089473";
```

Comment:
Is there a reason not to use a released version here? https://github.com/jonnenauha/prometheus_varnish_exporter/releases